gef-ui/server/utils/helpers.js

```diff
@@ -220,6 +223,8 @@ const summaryItemsConditions = (summaryItemsObj) => {
   const changedToIssueShow = id === 'name' || id === 'coverStartDate' || id === 'coverEndDate' || id === 'issueDate';
   // column key to display add if facility not yet issued
   const unissuedShow = id === 'hasBeenIssued';
+  // Issued facility change link (post confirmation)
+  const issuedHref = `/gef/application-details/${app._id}/${data.details._id}/confirm-cover-start-date`;
```

Comment:
Would be good to add a unit test for this, otherwise LGTM

Reply:
Added